roles/network_plugin/calico/tasks/main.yml

```diff
@@ -52,6 +42,16 @@
   changed_when: false
   when: "{{ use_hyperkube_cni|bool }}"
```

Comment:
We can use `rsync` instead of `cp -r` which will make the command itself idempotent. Of course, overwriting of CNI binaries will still be non-idempotent but that's a specific use case mostly for testing.